src/GeoPatchJobs.cpp

```diff
@@ -33,28 +33,23 @@ void SSingleSplitRequest::GenerateMesh() const
 	const int numBorderedVerts = borderedEdgeLen * borderedEdgeLen;
 
 	// generate heights plus a 1 unit border
-	std::vector<vector3d> positions;
-	positions.reserve(numBorderedVerts);
-	std::vector<double> heightsOut;
-	heightsOut.resize(numBorderedVerts);
+	vector3d *vrts = borderVertexs.get();
```

Comment:
Nitpick: as this pointer is mutated during the fill loop, I'd prefer it not be reused later in a separate loop, instead naming it something like `fill_verts`.